org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -249,29 +249,29 @@ private static Method getMethod(Class<?> clazz, String name) throws IllegalAcces
 		}
 		return null;
 	}
-	
 
-	
+
+
 	public boolean isModelField(String name) throws NoSuchFieldException, SecurityException {
         return isModelField(model, name);
 	}
 
 	public static boolean isModelField(Object model, String name) throws NoSuchFieldException, SecurityException {
-		
+
 		Field field = null;
 		Class<? extends Object> cls = model.getClass();
-    	while (!cls.equals(Object.class)) {
-    		try {
-    			field = cls.getDeclaredField(name);
-    			break;
-    		} catch (Exception ne) {
-    			cls = cls.getSuperclass();
-    		}
-    	}
+	while (!cls.equals(Object.class)) {
+		try {
+			field = cls.getDeclaredField(name);
+			break;
+		} catch (Exception ne) {
+			cls = cls.getSuperclass();
+		}
+	}
 
 		FieldDescriptor omf = field.getAnnotation(FieldDescriptor.class);
 		if (omf!=null && !omf.visible()) return false;
-		
+
 		final String getter = getGetterName(name).toLowerCase();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getGetterName" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)